values.yaml

```diff
@@ -156,6 +156,20 @@ global:
     # This setting must be true in both primary and secondary datacenters.
     enabled: false
 
+  # Configure ACLs.
+  acls:
+    # If true, an ACL token will be created that can be used in secondary
+    # datacenters for replication. This should only be set to true in the
+    # primary datacenter since the replication token must be created from that
+    # datacenter. It will then be imported into secondary datacenters.
```

Comment:
It'd be nice to point to `replicationToken` setting here and say that this is the value you should set in the secret for the replication token.